uaclient/tests/test_cli_enable.py

```diff
@@ -395,33 +441,36 @@ def test_entitlement_instantiated_and_enabled(
         self,
         _m_is_config_value_true,
         _m_get_available_resources,
-        silent_if_inapplicable,
+        _m_request_updated_contract,
+        m_getuid,
         allow_beta,
+        FakeConfig,
     ):
+        m_getuid.return_value = 0
         m_entitlement_cls = mock.Mock()
-        m_cfg = mock.Mock()
+        m_entitlement_obj = m_entitlement_cls.return_value
+        m_entitlement_obj.can_enable.return_value = (True, None)
+        m_entitlement_obj.enable.return_value = True
+
+        cfg = FakeConfig.for_attached_machine()
 
         m_ents_dict = {"testitlement": m_entitlement_cls}
 
-        kwargs = {"allow_beta": allow_beta}
-        if silent_if_inapplicable is not None:
-            kwargs["silent_if_inapplicable"] = silent_if_inapplicable
+        args = mock.MagicMock()
+        args.assume_yes = False
+        args.beta = allow_beta
+        args.service = ["testitlement"]
 
         with mock.patch.object(
             entitlements, "ENTITLEMENT_CLASS_BY_NAME", m_ents_dict
         ):
-            ret = _perform_enable("testitlement", m_cfg, **kwargs)
+            ret = action_enable(args, cfg)
 
         assert [
-            mock.call(m_cfg, assume_yes=False, allow_beta=allow_beta)
+            mock.call(cfg, assume_yes=False, allow_beta=allow_beta)
         ] == m_entitlement_cls.call_args_list
 
         m_entitlement = m_entitlement_cls.return_value
-        if silent_if_inapplicable:
-            expected_enable_call = mock.call(silent_if_inapplicable=True)
-        else:
-            expected_enable_call = mock.call(silent_if_inapplicable=False)
+        expected_enable_call = mock.call()
         assert [expected_enable_call] == m_entitlement.enable.call_args_list
-        assert ret == m_entitlement.enable.return_value
-
-        assert 1 == m_cfg.status.call_count
```

Comment:
Why are we dropping those asserts ?

Reply:
@lucasmoura Good catch! I must've dropped it while getting other asserts working and forgot to add it back. Just pushed and it should be back now